### PR TITLE
docs: Updated provider status in README.md

### PR DESCRIPTION
**Pull request by chandrasekharan-zipstack**
## What

- Updated the status of providers in `README.md`
- Added bedrock (working)
- Marked few "coming soon" to working"
- Added SFTP (coming soon)

## Why

- To reflect the current state of the system

## Can this PR break any existing features. If yes, please list possible items. If no, please explain why. (PS: Admins do not merge the PR without this section filled)

- No, doc change only

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,27 +102,29 @@ Unstract comes well documented. You can get introduced to the [basics of Unstrac
 |<img src="docs/assets/3rd_party/openai.png" width="32"/>| OpenAI | ✅ Working |
 |<img src="docs/assets/3rd_party/azure_openai.png" width="32"/>| Azure OpenAI | ✅ Working  |
 |<img src="docs/assets/3rd_party/palm.png" width="32"/>| Google PaLM | ✅ Working  |
+| <img src="docs/assets/3rd_party/ollama.png" width="32"/>| Ollama | ✅ Working |
```
Comment (gaya3-zipstack) [inline README.md:105]: Is there a spacing issue here?

Comment (chandrasekharan-zipstack) [reply README.md]: Shouldn't be a problem, however I've removed the extra space

**Review by hari-kuriakose — APPROVED** [review]
@chandrasekharan-zipstack LGTM overall

**Review by kirtimanmishrazipstack — APPROVED** [review]

**Review by kirtimanmishrazipstack — APPROVED** [review]

**Review by muhammad-ali-e — APPROVED** [review]
LGTM

**Pull request merged**